pages/common/scrapy.md

```diff
@@ -0,0 +1,31 @@
+# scrapy
+
+> Open Source framework for crawling websites written in python.
+
+- Create a project:
+
+`scrapy startproject {{project_name}}`
+
+- Create a spider:
+
+`scrapy genspider {{spider_name}} {{website_domain}}`
+
+- Edit spider:
+
+`scrapy edit {{spider_name}}`
+
+- Run spider:
+
+`scrapy crawl {{spider_name}}`
+
+- Fetch a webpage as scrapy sees it and print source in stdout:
+
+`scrapy fetch {{url}}`
+
+- View a webpage as scrapy sees it, opens it in browser(disable javascript before):
```

Comment:
@waldyrious js script must be disabled to have a proper view representation since scrapy doesn't render javascript. Even though it's not necessary enabled javascript would defeat the whole purpose of this command - to see what scrapy spider would see.